framework/requirements.txt

```diff
@@ -43,6 +43,7 @@ idna==3.7.0
 importlib-metadata==6.8.0
 inflection==0.3.1
 isodate==0.6.1
+itsdangerous==2.0.0
```

Comment:
```suggestion
```

I think this dep should not be included :thinking: It was removed in 4.9.0 when `connexion` was upgraded to 3.0.0:
- https://github.com/wazuh/wazuh/pull/23199/files#diff-ac0d503454a85f64c8021ded91db3e9afddcc51dd88864ca250ac0e199da797a 

Also, it is not being included in the deps 30 cpython:
![image](https://github.com/wazuh/wazuh/assets/23361101/bedbbc5d-6e38-471c-99bd-d127689df18c)